src/main.ts

```diff
@@ -1,7 +1,7 @@
 import { NestFactory } from '@nestjs/core';
-import { AppModule } from './app.module';
+import AppModule from './app.module';
 
-async function bootstrap() {
+async function bootstrap(): Promise<void> {
```

Comment:
`Promise<void>` doesn't bring any relevant & useful information here. I know that it's forced by the Style Guide, but I doubt we should follow this.